package.json

```diff
@@ -43,7 +43,7 @@
   },
   "dependencies": {
     "@hapi/joi": "^15.0.2",
-    "asl-validator": "^1.5.0",
+    "asl-validator": "github:aslafy-z/asl-validator#patch-1",
```

Comment:
Why did you use your patch one? It would had better to be merged it to asl-validator. Is it possible?

Reply:
I'm waiting for my other PR to be merged and released before I can switch back the the real semver. Added a WIP tag in the title.

Reply:
Thanks! Agreed with that 👍